<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAbout</class>
 <widget class="QDialog" name="DialogAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="IconView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>15</y>
     <width>454</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CWP Bible Study Program</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>95</y>
     <width>466</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>This application uses Qt Libraries under the LGPL v3 license</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>License is available at: http://www.gnu.org/licenses/lgpl.html</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblQtVersion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Qt Version: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCWPVersion">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>55</y>
     <width>466</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLicense">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>125</y>
     <width>466</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Licensed under GPL v1-3 (your choice)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
